lexibank_cosgrovevoro.py

```diff
@@ -1,81 +1,111 @@
-from pathlib import Path
+import attr
+from clldutils.path import Path
 import pylexibank
 
-# Customize your basic data.
-# if you need to store other data in columns than the lexibank defaults, then over-ride
-# the table type (pylexibank.[Language|Lexeme|Concept|Cognate|]) and add the required columns e.g.
-#
-#import attr
-#
-#@attr.s
-#class Concept(pylexibank.Concept):
-#    MyAttribute1 = attr.ib(default=None)
+
+@attr.s
+class IDSLexeme(pylexibank.Lexeme):
```

Comment:
Maybe we need to put this code - which should be shared across IDS datasets - into a pyids package.